transhumandesign/kag-base
# Translatable thing checking

## Status

- **READY**: this PR is (to the best of your knowledge) ready to be incorporated into the game.

## Description

When I'm rechecking jp translation file, I noticed some scripts are needed to change for translating.

## Steps to Test or Reproduce

See commit comment for check what strings are changed and what strings should translator translate.

## To all translators

Please translate them:
"Crate with {ITEM}"

"{COLOR}{NAME}{COLOR}\n\ntechnology required.\n"
Check Workbench.as for {NAME}:
-"Drill Technology"
-"Saw Technology"
-"Dinghy Technology"

"Pickup"
Names of production and pickup items, please check these words:
-"Water Arrows"(or forget to add "Water Arrow"?)
-"Bomb Arrows"
-"Bombs"
-"Water Bombs"
-"Seed"
-"Log"
-"Food"
Search with addScrollItemsToArray in WAR_Technology.as and PickupWheelMenuEntry in StandardPickup.as.

"Username copied: {NAME}"

Chat events, translate if chat font has characters for the language:
"{NAME} available."
"{TEAM}'s "
"{TEAM_S}flag has been picked up by {PLAYER}!"
"{TEAM_S}flag has been captured by {PLAYER}!"
"{TEAM_S}flag has been returned due to teamwork!"
"{TEAM_S}flag has been returned!"
